aggregator_core/src/datastore.rs

```diff
@@ -4453,12 +4470,26 @@ pub mod models {
             self.ord
         }
 
+        /// Returns the last preparation step returned by the Helper, if any.
```

Comment:
The Helper always uses this field to store the last preparation step it returned to the Leader, to allow replay to occur. (during processing of an aggregation init/continue message, the Helper updates its in-memory "last" prep step to be what it will eventually return, then builds its response message based on the "last" prep steps) The Helper doesn't need to store the Leader's last step.

(and, slightly off-topic from the question, the Leader always stores None in this field.)